tests/c/aot_debuginfo.c

```diff
@@ -1,4 +1,5 @@
-// ignore-if: test $YK_CARGO_PROFILE != "debug"
+// Compiler:
+//   env-var: YKT_EXTRA_CC_FLAGS=-O1 -g
```

Comment:
What does `YKT` mean?

Reply:
I *think* this should be `YKB_EXTRA_...`?

Reply:
`YKT` was traditionally used for testing only, but I see it's been renamed/removed at some point.

You can't use this variable outside of tests since it requires running the compiler via the compiler wrapper. Is that OK still?

Reply:
Should we just rename it to `YKB` since it's a build-time option (even if only "build testing")? If you think that's the way to go, you can rename + squash in one go.

Reply:
OK.

I must admit I get confused at times with our variables. I've seen so many reincarnations of them that I'm just constantly lost!

Reply:
I know the feeling! FWIW, I *think* the prefixes are now documented in the book tolerable well?

Reply:
Yeah mostly. There's a couple of things that often still catch me out:

 - variables with no `D` or `B`, like `YK_LOG`. I always type `YKD_LOG`.
 - the mandatory `<path>:` prefix of `YKD_LOG_IR`, because I was spoiled by the optional prefix of `YK_LOG` :)

Reply:
> the mandatory <path>: prefix of YKD_LOG_IR, because I was spoiled by the optional prefix of YK_LOG :)

Hmm, you're right, that shouldn't be mandatory! PR welcome!

Reply:
I know. It's just such a small fish right now!

Reply:
Anyway YKT -> YKB in [a04c8f8](https://github.com/ykjit/yk/pull/1411/commits/a04c8f8a0efdc5c2ed2fc5f20e9fa0ab36560fdf)